CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.11] - 2023-12-02
```

Comment:
Sorry, but I put my changelog entry for `Stopped` in the prior version again 🤦‍♂️ mind fixing that too before merging?

Reply:
Yep, done.